middleware/healthcheck/healthcheck.go

```diff
@@ -40,11 +41,19 @@ func New(config ...Config) fiber.Handler {
 			return c.Next()
 		}
 
-		switch c.Path() {
-		case cfg.ReadinessEndpoint:
-			return isReadyHandler(c)
-		case cfg.LivenessEndpoint:
-			return isLiveHandler(c)
+		prefixCount := len(utils.TrimRight(c.Route().Path, '/'))
+		if len(c.Path()) >= prefixCount {
+			checkPath := c.Path()[prefixCount:]
+			checkPathTrimmed := checkPath
+			if !c.App().Config().StrictRouting {
+				checkPathTrimmed = utils.TrimRight(checkPath, '/')
```

Comment:
For Strict Routing we are calling TrimRight twice. Once here and once in line 44

Reply:
right, because we want to subtract the slash once from the request path and once from the route path

Reply:
I see it now, i was confused for a second